.travis.yml

```diff
@@ -1,18 +1,18 @@
 # Config file for automatic testing at travis-ci.org
-
+dist: xenial
 sudo: false
```

Comment:
The __sudo:__ tag is now fully deprecated in Travis CI so all lines that start with __sudo:__ can safely be deleted.

Reply:
@cclauss is right but at the same time I hope to see it merged before travis deprecates another feature. Travis config was broken for ages...